lib/reporter.ts

```diff
@@ -275,7 +289,7 @@ class ReportPortalReporter extends Reporter {
     log.debug(`Runner start`);
     this.rpPromisesCompleted = false;
     this.isMultiremote = runner.isMultiremote;
-    this.sanitizedCapabilities = runner.sanitizedCapabilities;
+    this.suitesDescription.push(runner.sanitizedCapabilities)
```

Comment:
Let's not do it that way. sanitizedCapabilities is the browser, version, and platform - it seems to be a test item attribute rather than a description. If you need it in the description, you can add it to your project hooks using addDescriptionToAllSuites.

Reply:
I actually dont need this information in the description. But as it was there before, so I did not change it. I removed it now and restored the `sanitizedCapabilities` property as it gets also used in the `onTestStart` method.